icubam/db/store.py

```diff
@@ -463,11 +463,25 @@ def manages_user(self, manager_user_id: int, user_id: int) -> bool:
     """Returns true if the manager user can manage the user."""
     if self.is_admin(manager_user_id) or manager_user_id == user_id:
       return True
-    return self._session.query(
+
+    # TODO(olivier): for some reason, this query sometimes returns False
+    # while it should not.
+    result = self._session.query(
```

Comment:
Do you have an example? What we do below is essentially not different from this, but in a more convoluted way and I'm not sure if it will solve the issue.